src/interpreter/ByteCode.h

```diff
@@ -1205,11 +1220,11 @@ class GetObjectPreComputedCase : public ByteCode {
     union {
         GetObjectInlineCacheSimpleCase* m_simpleInlineCache;
         GetObjectInlineCacheComplexCase* m_complexInlineCache;
```

Comment:
Just a suggestion about naming.
The name of cache structures `GetObjectInlineCacheSimpleCase` and `GetObjectInlineCacheComplexCase` is somewhat confusing with the bytecode `GetObjectPreComputedCase`.
How about renaming these structures by adding `Data` suffix similar to the cache structure in property setting (`SetObjectInlineCacheData`)?

Reply:
That's a good idea!